data/p/slack-gitlab-merge-requests-integrations.mdx

```diff
@@ -31,6 +40,21 @@ Focusing on collaboration across the whole engineering team, Axolo is not anothe
 
 Axolo makes a temporary Slack channel for every GitLab merge request, it will invite reviewers and assignees to the channel, notify them of relevant events like code comments, comments, GitLab actions, deployments and finally it will archive the channel once you close or merge the request.
 
+#### GitLab CI Slack notification
+
+Axolo recently added support for GitLab CI. Now for every merge requests Axolo will send a summary when all the CI is finished to the relevant Slack channel. It will look like this:
```

Comment:
Je me suis permis de changer la tournure et ajouter les mots pipeline, build & deployment qui sont importants pour GitLab CI

```suggestion
Axolo recently added support for GitLab CI. How does it work? Axolo will send a summary of your jobs (builds or deployments) when a merge request pipeline is completed to the relevant Slack channel. It will look like this:
```